sycl/doc/syclcompat/README.md

```diff
@@ -1838,6 +1838,46 @@ inline constexpr RetT extend_max_sat(AT a, BT b, CT c,
                                      BinaryOperation second_op);
 ```
 
+The math header file provides APIs for bit-field insertion (`bfi_safe`) and bit-field extraction (`bfe_safe`):
+
+```c++
+
+/// Bitfield-insert with boundary checking.
+///
+/// Align and insert a bit field from \param x into \param y . Source \param
+/// bit_start gives the starting bit position for the insertion, and source
+/// \param num_bits gives the bit field length in bits.
+///
+/// \tparam T The type of \param x and \param y , must be an unsigned integer.
+/// \param x The source of the bitfield.
+/// \param y The source where bitfield is inserted.
+/// \param bit_start The position to start insertion.
+/// \param num_bits The number of bits to insertion.
```

Comment:
This is more for discussion than a request. So far the headers have not been including in the README the doxygen documentation. 
What do you think we should do here? 

The best is probably setting up doxygen as (I think you) already suggested. However, I am referring to this PR and the few upcoming APIs we have planned for now.

Reply:
I think we agreed for now that we can introduce docs like this for now, because soon we'll move to doxygen.